ConsoleApp/Tasks/CheckTernaryFormInIntArray.cs

```diff
@@ -0,0 +1,79 @@
+﻿/*Task 1. Create a console application, whichasks the user for two integers a and b (assume
+that the user enters integers without errors). The application then prints out all integers
+in the range from a (inclusive) to b (inclusive) that have exactly two 2's in their ternary
+number system (0, 1, 2) representation. Develop a console application that implements
+the specified functionality.*/
+
+public class CheckTernaryFormInIntArray
+{
+ static private string _string = string.Empty;
+
+ private static int ReturnArrayBeginning()
+ {
+  Console.WriteLine("Enter a number for array's beginning");
+  return Convert.ToInt32(Console.ReadLine());
+ }
+
+ private static int ReturnArrayEnd() 
+ {
+  Console.WriteLine("Enter a number for array's ending");
+  return Convert.ToInt32(Console.ReadLine());
+ }
+
+ private static string ReverseString(string parString)
+ {
+  char[] charArray = parString.ToCharArray();
+  Array.Reverse(charArray);
+  return new string(charArray);
+ }
+
+ static private string ConvertToTernary(int N)
+ {
+  string ternaryNumber = string.Empty;
+
+  if (N == 0)
+  {
+   ternaryNumber = "0";
+   return ternaryNumber;
+  }
+
+  while(N >= 1) 
+  {
+   ternaryNumber += N % 3;
+   N /= 3;
+  }
+
+  ternaryNumber = ReverseString(ternaryNumber);
+
+  return ternaryNumber;
+ }
+
+ static private bool IsHasTwoTwos(string parString)
+ {
+  int twoCount = 0;
+
+  for(int i=0; i<parString.Length; i++)
```

Comment:
Format the code using Visual Studio -> Edit -> Advanced -> Format Document feature. 

The spaces should be around =, < operators.

Reply:
Replace for cycle with foreach